src/iam/iam-base.ts

```diff
@@ -213,10 +213,8 @@ export class IAMBase {
         signer: this._signer,
         publicKey: this._publicKey
       });
-      this._publicKey = this._didSigner.publicKey;
```

Comment:
I think this should stay, that was added on purpose, cause after logging in the publicKey is stored only on session storage but not on the this._publicKey property, so when calling this._isSessionActive() it returns false.

Reply:
Right. I didn't noticed that pub key used to check session status. Will put it back